go.mod

```diff
@@ -3,52 +3,45 @@ module github.com/google/cadvisor
 go 1.13
 
 require (
-	cloud.google.com/go v0.26.0
+	cloud.google.com/go v0.51.0
 	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
-	github.com/Microsoft/go-winio v0.4.15-0.20190919025122-fc70bd9a86b5 // indirect
-	github.com/aws/aws-sdk-go v1.6.10
-	github.com/blang/semver v3.1.0+incompatible
-	github.com/cilium/ebpf v0.0.0-20200702112145-1c8d4c9ef775 // indirect
-	github.com/containerd/containerd v1.4.0-beta.2
-	github.com/containerd/ttrpc v1.0.1 // indirect
+	github.com/Microsoft/go-winio v0.4.15-0.20200908182639-5b44b70ab3ab // indirect
+	github.com/aws/aws-sdk-go v1.35.5
+	github.com/blang/semver v3.5.0+incompatible
+	github.com/containerd/containerd v1.4.1
+	github.com/containerd/ttrpc v1.0.2 // indirect
 	github.com/containerd/typeurl v1.0.1
-	github.com/coreos/go-systemd/v22 v22.1.0 // indirect
 	github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible // indirect
-	github.com/docker/docker v0.7.3-0.20190327010347-be7ac8be2ae0
+	github.com/docker/docker v17.12.0-ce-rc1.0.20200916142827-bd33bbf0497b+incompatible
 	github.com/docker/go-connections v0.4.0
```

Comment:
quick question here: you mentioned the goal is to upgrade to docker v19.03.13, but the version here is v17.12.0. I don't see any v19 here: https://pkg.go.dev/mod/github.com/docker/docker?tab=versions

I'm guessing there's some type of mapping I'm missing, can you please explain how that works?

Reply:
So pick the `v19.03.13` in docker/docker repo:
https://github.com/moby/moby/tree/v19.03.13

If you see the SHA it is:
`bd33bbf`

if you do a `go get` with the SHA, you will get the random looking version thingy!

Reply:
i know it's crazy! we've been doing this for a while